torchao/float8/float8_python_api.py

```diff
@@ -30,14 +30,19 @@ def addmm_float8_unwrapped(
     output_scale: Optional[torch.Tensor] = None,
     bias: Optional[torch.Tensor] = None,
     use_fast_accum: bool = False,
+    inverse_scale: bool = True
```

Comment:
does this mean `AffineQuantizedTensor` uses scale in the same way `Float8Tensor` uses 1 / scale, or something else?